datasets/ch/seco_sanctions/crawler.py

```diff
@@ -499,28 +498,44 @@ def parse_entry(context: Context, target: Element, programs, places):
                     context, prompt, value, OtherInfo, model=LLM_VERSION
                 )
                 review = request_review(
-                    context,
-                    value,
-                    value,
-                    PLAIN,
-                    "other-information",
-                    None,
-                    extraction,
-                    VERSION,
+                    context=context,
+                    key_parts=value,
+                    source_value=value,
+                    source_mime_type=PLAIN,
+                    source_label="other-information",
+                    source_url=None,
+                    orig_extraction_data=extraction,
+                    model_version=CRAWLER_VERSION,
                 )
 
         if review.accepted:
             for extracted_value in review.extracted_data.simple_values:
-                if "Date" in extracted_value.property:
-                    h.apply_date(
-                        entity, extracted_value.property, extracted_value.value
+                prop = entity.schema.get(extracted_value.property)
+                if prop is not None:
+                    if prop.type == registry.date:
+                        h.apply_date(
+                            entity, extracted_value.property, extracted_value.value
+                        )
+                        continue
+                    entity.add(
+                        extracted_value.property,
+                        extracted_value.value,
+                        origin=LLM_VERSION,
+                        original_value=value,
+                    )
+                else:
+                    if (
+                        extracted_value.property == "imoNumber"
+                        and not entity.schema.is_a("Vessel")
```

Comment:
When does this happen? At this point it's not a Vessel because otherwise we wouldn't have ended up in this branch of the if, right?